src/main/java/com/odysseusinc/athena/model/athena/DownloadShare.java

```diff
@@ -0,0 +1,64 @@
+package com.odysseusinc.athena.model.athena;
+
+
+import com.odysseusinc.athena.model.security.AthenaUser;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "download_share")
+public class DownloadShare {
+    @EmbeddedId
+    private DownloadShareId downloadShareId;
```

Comment:
There is an implicit linkage to `DownloadBundle`. But why not to refer directly to it?
And no problems with orphans, no issue with search query.

Reply:
fixed